src/components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './Header.module.css';
+
+const Header = () => (
+  <header>
+    <nav className={styles.nav}>
+      <h1>BookStore</h1>
+      <NavLink to="/books">Home</NavLink>
+      <NavLink to="/categories">Calculator</NavLink>
```

Comment:
- [ ] The text for this link should be Categories, there seems to be a typo. Kindly modify this.